src/azure-cli/azure/cli/command_modules/profile/custom.py

```diff
@@ -174,7 +174,7 @@ def login(cmd, username=None, password=None, service_principal=None, tenant=None
         raise CLIError(err)
     except requests.exceptions.SSLError as err:
         from azure.cli.core.util import SSLERROR_TEMPLATE
-        raise CLIError(SSLERROR_TEMPLATE.format(str(err)))
+        raise CLIError(SSLERROR_TEMPLATE + " Error detail: {}".format(str(err)))
```

Comment:
@houk-ms, let's keep the current behavior. I will adopt `azclierror` in Azure Identity integration.